cypress/e2e/CMS/$InfographicCreation/index.js

```diff
@@ -67,16 +67,20 @@ And('description reads {string}', (contentText) => {
     cy.get(`p:contains('${contentText}')`).should('be.visible');
 });
 
-And('the image has the name {string}', (imageName) => {
-    cy.get('div.centered-element source[media="(min-width: 769px)"]').should('have.attr', 'srcset').and('include', imageName);
-});
+ And('the image has the name {string}', (imageName) => {
+//     cy.get('div.centered-element source[media="(min-width: 769px)"]').should('have.attr', 'srcset').and('include', imageName);
```

Comment:
Is this deliberately commented out?  Or something you meant to restore after debugging?

If it's code that's no longer needed, delete it.  If you meant to restore it, then please do.

There should not be commented out code.